Prevent facter's build script from gzipping the manpages.  Our version
of man doesn't understand manpages with a .gz extension

--- facter-1.6.16/install.rb.orig	2013-01-04 14:11:28.431143114 -0700
+++ facter-1.6.16/install.rb	2013-01-04 14:11:35.402916848 -0700
@@ -107,9 +107,6 @@
     FileUtils.makedirs(om, {:mode => 0755, :verbose => true})
     FileUtils.chmod(0755, om)
     FileUtils.install(mf, omf, {:mode => 0644, :preserve => true, :verbose => true})
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
   end
   else
   puts "Skipping Man Page Generation"
